### ISSUE TITLE
Using deprecated `torch.norm` to compute loss

### ISSUE THREAD

**Issue by oliverwm1** [ai2cm/ace#223]
We use `torch.norm` in computing loss but it is [deprecated](https://pytorch.org/docs/1.13/generated/torch.norm.html?highlight=norm#torch.norm).

https://github.com/ai2cm/full-model/blob/fe57d295dca1575ddebe69a91a8421cef2c1434b/fme/fme/core/loss.py#L124-L127

**Comment by mcgibbon** [comment]
@gideonite also pointed out that torch.norm is a very strange choice for loss, it's the square root of the sum of squared errors.

**Comment by oliverwm1** [comment]
ai2cm/full-model#414 added the capability to use MSE-based loss function and avoid `torch.norm`. But we need to update our configs to use this option and ensure that performance is similar to current loss function.

**Comment by gideonite** [comment]
While working on weight decay, it started to make a bit more sense why the outputs are normalized. This makes the loss functions more comparable across resolutions (e.g. 4º to 1º)
